app/my_jss/index.js

```diff
@@ -0,0 +1,180 @@
+
+
+function loginFunction(){
+  window.location.href = "login.html";
+}
+
+var dataInformation = JSON.parse(playlistsJSON);
+var allPlaylist = document.getElementsByClassName('section-absolute')[0];
+var popUpPlaylist = document.getElementById('playlistAbsolut');
+
+
+
+generatePlaylists();
+
+
+function generatePlaylists(){
+
+for (var i=0; i < dataInformation.length; i++)
+  {
+      createElement(dataInformation[i]);
+  }
+}
+function createElement(elem){
+  var bigDiv = document.createElement('div');
+  bigDiv.setAttribute("class","individual-playlist");
+  allPlaylist.appendChild(bigDiv);
+  var middleDiv= document.createElement('div');
+  middleDiv.setAttribute("id","individual-playlist-button");
+  //ADD HERE THE EVENT
+  bigDiv.appendChild(middleDiv);
+
+  middleDiv.addEventListener('click', function(){
+    var title = elem.title;
+    showPlaylist(title);
+  });
+  /////
+  var imgBut = document.createElement('img');
```

Comment:
Split this method in 4-5  smaller methods: eg drawSongImage etc. Don't name the method createElement because it's the same name as a native API and it doesn't say anything about what the function does.
You should really pay a lot more attention to the variables/function names you are using